.codeclimate.yml

```diff
@@ -47,27 +34,6 @@ engines:
       languages:
         - ruby
         - javascript
-
-# Engines can analyze files and report issues on them, but you can separately
-# decide which files will receive ratings based on those issues. This is
-# specified by path patterns under the ratings key.
-
-# For more details see here:
-# http://docs.codeclimate.com/article/289-configuring-your-repository-via-codeclimate-yml#platform
-
-# Note: If the ratings key is not specified, this will result in a 0.0 GPA on your dashboard.
-
-ratings:
-  paths:
-    - "Gemfile.lock"
-    - "app/javascript/**/*.jsx"
-    - "app/**/*.{rb,js,es6,coffee,scss,css}"
-    - "lib/**/*.{rb,js,es6,coffee}"
-    - "test/**/*.rb"
-
-# You can globally exclude files from being analyzed by any engine using the
-# exclude_paths key.
-
 exclude_paths:
```

Comment:
What about `exclude_paths` has been changed to `exclude_patterns`?

Reply:
Right, good catch!